x/programs/rust/examples/nft/src/lib.rs

```diff
@@ -159,16 +158,23 @@ pub fn burn(program: Program, from: Address) -> bool {
     // Burn the NFT by transferring it to the zero address
     program
         .state()
-        .store(StateKey::Balance(from).to_vec(), &(balance - BURN_AMOUNT))
+        .store(StateKey::Balances(from).to_vec(), &(balance - BURN_AMOUNT))
```

Comment:
You should delete keys that no longer have a balance.

Reply:
There is no exposed functionality in state.rs to delete keys currently. cc @hexfusion

Reply:
Yeah anything you come across like this feel free to create an issue, and if you want, implement. I think one big reason we are doing these examples is to help flesh out missing pieces.